pkg/util/testing/v1beta2/wrappers.go

```diff
@@ -935,9 +935,26 @@ func (c *ClusterQueueWrapper) ResourceGroup(flavors ...kueue.FlavorQuotas) *Clus
 	return c
 }
 
-// AdmissionChecks replaces the queue additional checks
+// AdmissionChecks replaces the queue additional checks.
+// This is a convenience wrapper that converts to the AdmissionChecksStrategy format.
 func (c *ClusterQueueWrapper) AdmissionChecks(checks ...kueue.AdmissionCheckReference) *ClusterQueueWrapper {
-	c.Spec.AdmissionChecks = checks
+	if len(checks) == 0 {
+		c.Spec.AdmissionChecksStrategy = nil
+		return c
+	}
+
+	if c.Spec.AdmissionChecksStrategy == nil {
+		c.Spec.AdmissionChecksStrategy = &kueue.AdmissionChecksStrategy{}
+	}
+
```

Comment:
```suggestion
  c.Spec.AdmissionChecksStrategy = &kueue.AdmissionChecksStrategy{}
```
There should be no need to care about the corner cases in the testing wrappers. Also in the previous version we wouldn't care about them